scripts/setup.sh

```diff
@@ -182,6 +182,9 @@ GCP_ACCOUNT=$(gcloud config get-value account)
 echo -e "You are now logged in as [$GCP_ACCOUNT]"
 echo
 DEFAULT_PROJECT=$(gcloud config get-value project)
+GCP_REGION=$(gcloud config get-value functions/region)
```

Comment:
@mmajcher you are adding this in other PR as parameter right?

Reply:
Yeah as a yaml parameter.
As I understand Wlad only needs to read this value, as such it will be easy for me to replace this code with my variable. So can stay as is and I will update this later in my PR